.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I was looking at how other repos do this, and I really like Vercel's template as a form. Looks like Github support it via yaml. Can we do this instead? https://github.com/vercel/next.js/blob/canary/.github/ISSUE_TEMPLATE/1.bug_report.yml